src/App/Repository/ExportJobRepository.php

```diff
@@ -73,11 +73,9 @@ public function setSearchConditions(SearchData $search)
             $query->where('hxl_meta_data_id', '=', $search->hxl_meta_data_id);
         }
 
-        // Limit search to user's records unless they are admin
-        // or if we get user=me as a search param
-        if (! $this->isUserAdmin($user) || $search->user === 'me') {
-            $search->user = $this->getUserId();
-        }
+        // get user ID so that we only ever get jobs from that user
+        $search->user = $this->getUserId();
```

Comment:
Just wondering if there's any reason to check the query string, rather than just getting the logged in user? The original test had a query string with a user ID.

Reply:
We should always get the logged in user not trust a query string, though conceivably an Admin could run a request for a user's stuff and in that case we would care about the query string, however, that's not a feature we support at the moment.